src/utils.js

```diff
@@ -310,7 +311,26 @@ function getDayForMenu() {
   return today;
 }
 
-export function getTodaysPrestoMenu(menu) {
+export async function getMenu(link, parseMenu, allergens) {
+  const block = '```';
+  try {
+    const body = await request(link);
+    return `${block}${parseMenu(body, allergens)}${block}`;
+  } catch (e) {
+    logger.error(e);
+    return `${block}Chyba počas načítavania menu :disappointed:${block}`;
+  }
+}
+
+export async function getAllMenus(allergens) {
```

Comment:
do the requests in parallel
`const [presto, veglife, mizza] = await Promise.all([getMenu(...), getMenu(..), getMenu(...)])`